versions/next/asyncapi.md

```diff
@@ -618,14 +618,11 @@ This object can be extended with [Specification Extensions](#specificationExtens
   ],
   "message": {
     "headers": {
-      "type": "object",
-      "properties": {
-        "qos": {
-          "$ref": "#/components/schemas/MQTTQoSHeader"
-        },
-        "retainFlag": {
-          "$ref": "#/components/schemas/MQTTRetainHeader"
-        }
+      "qos": {
```

Comment:
QoS and retainFlag are MQTT headers. We should get rid of them here.

Reply:
Sure - will change and resubmit